LoopKit/Insulin/InsulinModelSettings.swift

```diff
@@ -10,13 +10,26 @@ public enum InsulinModelSettings: Equatable {
     case walsh(WalshInsulinModel)
 
     public static let validWalshModelDurationRange = TimeInterval(hours: 2)...TimeInterval(hours: 8)
+    
+    public var longestEffectDuration: TimeInterval {
+        return .hours(6) + .minutes(10)
```

Comment:
I'm guessing this is an edgecase (and not many people use Walsh), but I'm wondering if it could make sense to unwrap the enum and use the DIA of the model to compute `longestEffectDuration` to make sure we're not cutting off insulin activity prematurely for Walsh models.

Reply:
At some point we should probably drop Walsh support. But yes, I think basing this on the actual models that this factory can produce would be correct. This is a shortcut, and I'll update this.